adafruit_is31fl3731.py

```diff
@@ -413,3 +413,68 @@ def pixel_addr(x, y):
             x = x - 8
             y = y - 8
         return x * 16 + y
+
+
+class LedShim(Matrix):
+    """Supports the LED SHIM by Pimoroni"""
+
+    width = 28
+    height = 3
+
+
+    def __init__(self, i2c, address=0x75):
+        super().__init__(i2c, address)
+
+
+    # pylint: disable-msg=too-many-arguments
+    def pixelrgb(self, x, r, g, b, blink=None, frame=None):
+        """
+        Blink or brightness for x-pixel
+
+        :param x: horizontal pixel position
+        :param r: red brightness value 0->255
+        :param g: green brightness value 0->255
+        :param b: blue brightness value 0->255
+        :param blink: True to blink
+        :param frame: the frame to set the pixel
+        """
+        super().pixel(x, 0, r, blink, frame)
+        super().pixel(x, 1, g, blink, frame)
+        super().pixel(x, 2, b, blink, frame)
+
+
+
+    @staticmethod
+    def pixel_addr(x, y):
+        """Translate an x,y coordinate to a pixel index."""
+        lookup = [
```

Comment:
please at least make this a tuple, or a bytearray, - at best please algorithize as this table will take up valuable ram on small devices

Reply:
That is the table extracted from Pimoroni Python driver. I will see if I can find the logic and a few formula to get to the same result.

Reply:
great!!

Reply:
Done. No more table, just a list of test and basic math.